app/models/validation.rb

```diff
@@ -0,0 +1,10 @@
+class Validation
+  class << self
+    # Translated to human language:
+    # Match string that looks like <numbers> <dot> <numbers> or <numbers>
+    # Also means <dot> <numbers> won't evaluate to true
+    def is_numerical?(string)
```

Comment:
I think this could be simplified by trying to cast the string to a float. `Float(string)` rescue any potential error with `false`.